app/src/main/java/com/cornellappdev/uplift/ui/components/general/TabContentSelector.kt

```diff
@@ -82,7 +83,7 @@ fun TabContentSelector(tabs: List<Pair<String, @Composable () -> Unit>>) {
                             .fillMaxHeight()
                             .clickable(
                                 interactionSource = interaction,
-                                indication = rememberRipple()
+                                indication = ripple()
```

Comment:
Does this make a difference?

Reply:
yes the code will not compile bc rememberRipple is deprecated